toplevel/ccompile.ml

```diff
@@ -20,6 +20,15 @@ let create_empty_file filename =
   let f = open_out filename in
   close_out f
 
+(* Helper to time begin/end *)
+let time ~opts ~f ~header ~loc x =
+  if opts.config.time then
+    let vernac = Vernacexpr.({ expr = VernacComments [CommentString header]; control = []; attrs = []}) in
+    let vernac = CAst.make ~loc vernac in
+    let header = Topfmt.pr_cmd_header vernac in
```

Comment:
Shouldn't we expose some `pr_gen_header` which doesn't need to go through a vernacexpr instead?

Reply:
We could, but I deemed it not worth the effort.

Reply:
It bugs me so I pushed a commit

Reply:
Thanks!